cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConfiguration.java

```diff
@@ -5,37 +5,24 @@
 
 package io.strimzi.operator.cluster.model;
 
-import io.vertx.core.json.JsonObject;
-
 import java.util.List;
+import java.util.Map;
 
 import static java.util.Arrays.asList;
 
 /**
  * Class for handling Kafka configuration passed by the user
  */
 public class KafkaConfiguration extends AbstractConfiguration {
+
+    public static final String FORBIDDEN_PREFIXES = "listeners, advertised., broker., listener., host.name, port, "
```

Comment:
Why this change? It seems to make it quite hard to read. And why just here and not in `KafkaConnectConfioguration`?

Reply:
That sounds reasonable. Thanks.